snuba/web/views.py

```diff
@@ -302,26 +309,35 @@ def storage_delete(
     if http_request.method == "DELETE":
 
         check_shutdown({"storage": storage.get_storage_key()})
-
-        if get_config("storage_deletes_enabled", 0):
-            raise Exception("Deletes not enabled")
-
         body = parse_request_body(http_request)
-        delete_settings = storage.get_deletion_settings()
-
-        if not delete_settings.is_enabled:
-            raise Exception(
-                f"Deletes not enabled for {storage.get_storage_key().value}"
+        try:
+            """
+            columns is a key value mapping
+            of storage-column-name to a list of values
+            ex:
+            {
+                "id": [1, 2, 3]
+                "status": ["failed"]
+            }
+            which represents
+            WHERE id in (1,2,3) AND status='failed'
+            """
+            columns = body["columns"]
+        except Exception:
+            return make_response(
+                jsonify(
+                    {
+                        "error": "required input 'conditions' not present in body of request"
```

Comment:
> "required input 'conditions' not present in body of request"

I'm assuming you wanted `'columns'` not `'conditions'`

Reply:
yes good catch